### PR TITLE
Update sdk-update-api-spec.yaml workflow so that it can label its prs as 'enhancement'

### LINKED ISSUE #608: Automatically add labels to openapi update pull requests
The GH actions workflow that creates the OpenAPI schema update pull requests should automatically add "enhancement" labels to the pull requests so it doesn't need to be done manually. Example PR #603.

### PR DESCRIPTION
**Pull request by alialiusefi**
Closes #608 

Added a github action that will react to the other scheduled github action that updates open apis. It will add a label 'enhancement'.

### CODE REVIEW

**Comment by alialiusefi** [comment]
The github action technote-space/create-pr-action doesn't offer an out of the box solution on adding a label when createing a pr. Also, it doesnt return the result of the opened pr, such as the pr_id for me to label in the same workflow.
As an alternative, I decided to create another workflow. Hopefully this solution is ok.

**Comment by nielsvanvelzen** [comment]
I would prefer it to be in the same workflow, if we'd need to switch the PR create action to a different one that's fine.

**Comment by alialiusefi** [comment]
Updated existing sdk-update-api-spec.yaml workflow, it now uses peter-evans/create-pull-request@v4 to create prs instead.

File: .github/workflows/sdk-update-api-spec.yaml [hunk 1]
```diff
@@ -24,25 +24,30 @@ jobs:
...
-          PR_BRANCH_PREFIX: 'openapi-update-'
-          PR_BRANCH_NAME: '${PR_ID}'
-          PR_TITLE: 'Update OpenAPI to ${{ env.STABLE_API_VERSION }}'
-          PR_BODY: |
+          token: ${{ secrets.JF_BOT_TOKEN }}
+          commit-message: 'Update generated sources to ${{ env.STABLE_API_VERSION }}'
+          committer: 'jellyfin-bot'
+          author: 'team@jellyfin.org'
+          title: 'Update OpenAPI to ${{ env.STABLE_API_VERSION }}'
+          labels: enhancement
```
Comment (nielsvanvelzen) [inline .github/workflows/sdk-update-api-spec.yaml:44]: Can you use the following labels instead?

- [dependencies](https://github.com/jellyfin/jellyfin-sdk-kotlin/labels/dependencies)
- [openapi](https://github.com/jellyfin/jellyfin-sdk-kotlin/labels/openapi)

**Review by nielsvanvelzen — DISMISSED** [review]
The changes look fine, testing GH actions is a pain though so let's hope it works as expected when the next server version releases 😄

File: .github/workflows/sdk-update-api-spec.yaml (hunk 1 above)
Comment (alialiusefi) [reply .github/workflows/sdk-update-api-spec.yaml]: Sure, I replaced 'enhancement' label with 'dependencies' and 'openapi' labels. You might need to reapprove the MR.

**Review by nielsvanvelzen — APPROVED** [review]

**Pull request merged**